chia/wallet/wallet_state_manager.py

```diff
@@ -679,6 +698,113 @@ async def determine_coin_type(
 
         return None
 
+    async def auto_claim_coins(self) -> None:
+        # Get unspent clawback coin
+        current_timestamp = self.blockchain.get_latest_timestamp()
+        clawback_coins: Dict[Coin, ClawbackMetadata] = {}
+        tx_fee = uint64(self.config.get("auto_claim", {}).get("tx_fee", 0))
+        min_amount = uint64(self.config.get("auto_claim", {}).get("min_amount", 0))
+        unspent_coins = await self.coin_store.get_coin_records(
```

Comment:
this could return a very long list, right? it seems this list might not even fit in RAM if you have a big wallet

Reply:
good point, created a separate issue

Reply:
We don't have pagination for any spending use case. If this is a concern, then we also need to fix the standard wallet and cat wallet. We fetch all unspent coins and load them to the memory too. The number of Clawback coins is much less than the XCH or CAT coins, usually.